docs/source/tutorials/image_quality.rst

```diff
@@ -0,0 +1,79 @@
+And even though OV9782 has quite large pixels, in general the noise levels of global shutters are more significant than for
+rolling shutter.
+
+Low-light increased sensitivity
+###############################
+
+On the image below you can see how different sensitivity values affect the IQ. Sensitivity will only add analog gain, which
+will increase the image noise. When in low-light environment, one should always increase exposure, not sensitivity. Note
+that by default, depthai will always do so - but when using 30FPS, max exposure is 33ms. For the right image below, we
+have set ColorCamera to 10 FPS, so we were able to increase exposure to 100ms.
+
+.. image:: https://user-images.githubusercontent.com/18037362/181298535-085d135e-5817-48fa-8392-f711ade69a77.png
+
+Zoomed-in image of a standard A4 camera tuning target at 420cm (40 lux). We have used 12MP IMX378 (on OAK-D) for this image.
+
+Camera tuning
+#############
+
+Our library supports setting camera IQ tuning blob, which would be used for all cameras.
+By default, cameras will use a *general* tuning blob, which works great in most cases - so changing the camera
+tuning blob is not needed for most cases.
+
+.. code-block:: python
+
+    import depthai as dai
+
+    pipeline = dai.Pipeline()
+    pipeline.setCameraTuningBlobPath('/path/to/tuning.bin')
+
+**Available tuning blobs**
+
+To tune your own camera sensors, one would need Intel's software, for which a license is needed
+- so the majority of people will only be able to use pre-tuned blobs. Currently available tuning blobs:
+
+- **Mono tuning for low-light environments** `here <https://artifacts.luxonis.com/artifactory/luxonis-depthai-data-local/misc/tuning_mono_low_light.bin>`__. This allows auto-exposure to go up to 200ms (otherwise limited with default tuning to 33ms). For 200ms auto-exposure, you also need to limit the FPS (:code:`monoRight.setFps(5)`)
+- **Color tuning for low-light environments** `here <https://artifacts.luxonis.com/artifactory/luxonis-depthai-data-local/misc/tuning_color_low_light.bin>`__. Comparison below. This allows auto-exposure to go up to 100ms (otherwise limited with default tuning to 33ms). For 200ms auto-exposure, you also need to limit the FPS (:code:`rgbCam.setFps(10)`). *Known limitation*: flicker can be seen with auto-exposure over 33ms, it is caused by auto-focus working in continuous mode. A workaround is to change from CONTINUOUS_VIDEO (default) to AUTO (focusing only once at init, and on further focus trigger commands): :code:`camRgb.initialControl.setAutoFocusMode(dai.CameraControl.AutoFocusMode.AUTO)`
+- **OV9782 Wide FOV color tuning for low-light environments** `here <https://artifacts.luxonis.com/artifactory/luxonis-depthai-data-local/misc/tuning_color_ov9782_wide_fov.bin>`__.  Fixes lens color filtering at low light, see `blog post here <https://www.luxonis.com/blog/lens_color_filtering_enhancement>`__.
```

Comment:
Actually this OV9782 tuning mostly helps with this issue: https://www.luxonis.com/blog/lens_color_filtering_enhancement -- would be good to mention.
And it also improves LSC (Lens Shading Correction) in general, for wide FoV devices only at the moment. The limitation is that OV9282 operation isn't supported yet at the same time with OV9782 using this blob, we'll fix and integrate it into the firmware later.